common/app/services/RedirectService.scala

```diff
@@ -120,7 +122,7 @@ class RedirectService(implicit executionContext: ExecutionContext) extends GuLog
   def lookupRedirectDestination(source: String): Future[Option[Destination]] = {
     val fullUrl = if (source.head == '/') expectedSourceHost + source else s"$expectedSourceHost/$source"
     ScanamoAsync(DynamoDB.asyncClient)
-      .exec(table.get("source" -> fullUrl))
+      .exec(table.get("source" === fullUrl))
```

Comment:
Gets rid of error:

```
/Users/Ioanna_Kokkini/code/frontend/common/app/services/RedirectService.scala:125:32: method stringTupleToUniqueKey in object syntax is deprecated (since 1.0): use `attr === value` syntax
[error]       .exec(table.get("source" -> fullUrl))
[error]                                ^
```